tests/common/dualtor/dual_tor_io.py

```diff
@@ -180,7 +192,7 @@ def generate_from_t1_to_server(self):
             tcp_tx_packet = testutils.simple_tcp_packet(
                 eth_dst=eth_dst,
                 eth_src=eth_src,
-                ip_dst=ip_dst,
+                ip_dst=random.choice(self.server_ip_list),
```

Comment:
Can we still allow the caller to specify a specific destination port/server?

Reply:
Is there a case where we want to test for the traffic to specific server(s)? Although it may be good to have the ability, but I think we might never use it.

Reply:
I think so, since we want to test specific switchover scenarios/behaviors. It would be easier to simulate these scenarios for a single port rather than all ports at once (e.g. link down for a specific interface, rather than all at once)

Reply:
Added the argument support for specifying destination IPs.